.github/workflows/peek_icons.yml

```diff
@@ -38,7 +38,6 @@ jobs:
       - name: Run icomoon_peek.py
         env:
           PR_TITLE: ${{ github.event.pull_request.title }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why do we need to remove this btw?

Reply:
The token's not even needed. The script was not configured properly, it should be fixed now, though.